### PR TITLE
create lw13 hackathon event

### PR DESCRIPTION
**Pull request by craigcannon**

### CODE REVIEW

File: apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+title: 'Supabase Launch Week 13 Hackathon'
+meta_title: 'Supabase Launch Week 13 Hackathon'
+subtitle: 'Build an Open Source Project over 10 days. 5 prize categories.'
+meta_description: 'Build an Open Source Project over 10 days. 5 prize categories.'
+type: 'hackathon'
+onDemand: false
+date: '2024-12-05T09:00:00.000-07:00'
```
Comment (w3b6x9) [inline apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx:8]: dec 6?

```suggestion
date: '2024-12-06T09:00:00.000-07:00'
```

File: apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  - hackathon
+main_cta:
+  {
+    url: 'https://hackathon.dev/lw13',
+    target: '_blank',
+    label: 'Submit project'
+  }
+---
+
+The hackathon starts on Friday, December 5th at 09:00 am PT and ends on Sunday, December 15th at 11:59 pm PT. You could win an extremely limited edition Supabase swag and add your name to the Supabase Hackathon Hall of Fame.
```
Comment (w3b6x9) [inline apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx:21]:
```suggestion
The hackathon starts on Friday, December 6th at 09:00 am PT and ends on Sunday, December 15th at 11:59 pm PT. You could win an extremely limited edition Supabase swag and add your name to the Supabase Hackathon Hall of Fame.
```

File: apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+For some inspiration, check out all the [winners from previous hackathons](https://supabase.com/blog/tags/hackathon).
+
+This is the perfect excuse to "Build in a weekend, scale to millions.” Since you retain all the rights to your submissions, you can use the hackathon as a launch pad for your new Startup ideas, side projects, or indie hacks.
+
+## Key Facts
+
+- You have 10 days to build a new **Open Source** project using Supabase in some capacity
+  - Starting 09:00 am PT Friday, December 5th 2024
```
Comment (w3b6x9) [inline apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx:30]:
```suggestion
  - Starting 09:00 am PT Friday, December 6th 2024
```

File: apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+## Rules
+
+- Team size 1-4 (all team members on winning teams will receive a prize)
+- You cannot be on multiple teams
+- One submission per team
+- It's not a requirement to use AI
+- All design elements, code, etc., for your project must be created **during** the event
+- All entries must be Open Source (link to source code required in entry)
+- Must use Supabase in some capacity
+- Can be any language or framework
```
Comment (w3b6x9) [inline apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx:77]:
```suggestion
- Can use any language or framework
```

File: apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+## Prizes
+
+There are 5 categories there will be prizes for:
+
+- Best overall project (Swag kit)
+- Best use of AI (Swag kit)
+- Most fun / best easter egg (Swag kit)
+- Most technically impressive (Swag kit)
+- Most visually pleasing (Swag kit)
```
Comment (dshukertjr) [inline apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx:48]:
```suggestion
- Best overall project (Swag kit)
- Best use of AI (Swag kit)
- Most fun / best easter egg (Swag kit)
- Most visually pleasing (Swag kit)
- Best project built with bolt.new (Swag kit)
```

**Review by fsansalvadore — APPROVED** [review]

File: apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx (hunk 5 above)
Comment (w3b6x9) [reply apps/www/_events/2024-12-05-launch-week-13-hackathon.mdx:43]: taken care of here: https://github.com/supabase/supabase/pull/30973

**Pull request merged**